src/sage/rings/invariants/invariant_theory.py

```diff
@@ -533,7 +533,7 @@ def __init__(self, n, d, polynomial, *args, **kwds):
         elif len(variables) == n-1:
             variables = variables + (None,)
         else:
-            raise ValueError('need '+str(n)+' or '+
+            raise ValueError('need '+str(n)+' or ' +
                              str(n-1)+' variables, got '+str(variables))
         ring = polynomial.parent()
```

Comment:
There are many `+` here.

Reply:
the script "autopep8" is only fixing E225, namely when there is a space before but not after, or conversely

Reply:
I see. But I am not sure if leaving other pluses without spaces around is an improvement.

I mean, in general, all or nothing would be better than mixing things.

Reply:
In my opinion, this is not worse than it was.

Reply:
I have a different opinion: visually it looks worse than before.

Reply:
But okay if you insist. Anyway it is trivial.